server/src/repositories/media.repository.ts

```diff
@@ -83,18 +96,18 @@ export class MediaRepository implements IMediaRepository {
           width: stream.width || 0,
           codecName: stream.codec_name === 'h265' ? 'hevc' : stream.codec_name,
           codecType: stream.codec_type,
-          frameCount: Number.parseInt(stream.nb_frames ?? '0'),
-          rotation: Number.parseInt(`${stream.rotation ?? 0}`),
+          frameCount: Number.parseInt((options?.countFrames ? stream.nb_read_packets : stream.nb_frames) ?? '0') || 0,
+          rotation: Number.parseInt(`${stream.rotation ?? 0}`) || 0,
           isHDR: stream.color_transfer === 'smpte2084' || stream.color_transfer === 'arib-std-b67',
-          bitrate: Number.parseInt(stream.bit_rate ?? '0'),
+          bitrate: Number.parseInt(stream.bit_rate ?? '0') || 0,
         })),
       audioStreams: results.streams
         .filter((stream) => stream.codec_type === 'audio')
         .map((stream) => ({
           index: stream.index,
           codecType: stream.codec_type,
           codecName: stream.codec_name,
-          frameCount: Number.parseInt(stream.nb_frames ?? '0'),
+          frameCount: Number.parseInt((options?.countFrames ? stream.nb_read_packets : stream.nb_frames) ?? '0') || 0,
```

Comment:
this same logic exists in 4 places now. might as well extract it to something like
```ts
const parse = (value?: string | number) => Number.parseInt(value ?? 0) || 0;
```